src/screens/HomeScreen/components/SearchCarousel/SearchCarouselItem/SearchCarouselItem.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {Image, StyleSheet, Text, View} from 'react-native';
+import {Button, CustomIconNames} from '../../../../../components';
+import {
+  DeviceUtils,
+  horizontalScale,
+  verticalScale,
+} from '../../../../../utils';
+import {SearchCarouselItemProps} from './SearchCarouselItem.types';
+import {useThemeColor} from '../../../../../hooks';
+import {fontStyles} from '../../../../../constants';
+
+const SearchCarouselItem = (props: SearchCarouselItemProps) => {
+  const {colors} = useThemeColor();
+  return (
+    <View style={styles.rootContainer}>
+      <Image
+        source={{uri: props.image}}
+        style={styles.image}
+        resizeMode="stretch"
+      />
+      <View style={styles.overflowContainer}>
+        <Text style={[{color: colors.text2}, fontStyles.h1_Bold]}>
+          {props.name}
+        </Text>
+        <View style={styles.buttonContainer}>
```

Comment:
Why do we need this wrapper? Cant we give marginTop to Button style?

Reply:
Removed and added to button styles